extensions/wikia/AdEngine/.npmrc

```diff
@@ -1,3 +1,3 @@
 loglevel="warn"
 save-exact=true
-registry=https://artifactory.wikia-inc.com/artifactory/api/npm/wikia-npm/
+@wikia:registry=https://artifactory.wikia-inc.com/artifactory/api/npm/wikia-npm/
```

Comment:
Right now npm freezes during package resolving when all packages are fetched from artifactory. Let's minimize this number to necessary packages.